### PR TITLE
Cleanup config to reduce merge conflicts

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| BC breaks        | config => see upgrade.mc
| Documentation PR | none

### CODE REVIEW

**Comment by danrot** [comment]
@chirimoya Would you also like to have a look? I think this is something critical.

File: app/config/sulu.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                dir: "%kernel.root_dir%/../vendor/gedmo/doctrine-extensions/lib/Gedmo/Tree/Entity"
+                alias: "GedmoTree" # this one is optional and will default to the name set for the mapping
+                is_bundle: false
+        resolve_target_entities:
+            Sulu\Component\Security\Authentication\UserInterface: Sulu\Bundle\SecurityBundle\Entity\User
+            Sulu\Component\Security\Authentication\RoleInterface: Sulu\Bundle\SecurityBundle\Entity\Role
+            Sulu\Bundle\MediaBundle\Entity\CollectionInterface: Sulu\Bundle\MediaBundle\Entity\Collection
+            Sulu\Bundle\ContactBundle\Entity\AccountInterface: Sulu\Bundle\ContactBundle\Entity\Account
+
+# doctrine cache providers
```
Comment (chirimoya) [inline app/config/sulu.yml]: Doctrine

**Comment by chirimoya** [comment]
ack

**Comment by wachterjohannes** [comment]
fixed the typo

**Pull request merged**